ui/app/AppLayouts/Chat/controls/community/CommunityIntroMessageInput.qml

```diff
@@ -15,11 +15,11 @@ StatusInput {
     label: qsTr("Community introduction and rules")
     charLimit: 1400
 
-    input.multiline: true
-    minimumHeight: 400
-    maximumHeight: 400
+    multiline: true
+    minimumHeight: height
+    maximumHeight: (height - 30)
```

Comment:
What and why is 30?

Reply:
@osmaczko here we just need the input to follow its parent's height, so minimum and maximum height in this case is the parent's height. We don't need any further calculations, ain't we? -30 is the pixels difference for the bottom label which is visible, otherwise the text input together with its bottom label are going all the way down. I don't see any confusion here or any binding loops (you can also eventually run the code and let me know if you got anything).

Reply:
In this case, minimum and maximum height should not be set at all. Similarly as:
```
Layout.maximumHeight: height
Layout.minimumHeight: height
```
it does not make any sense.

"~30" pixels is a magic number and just asks for trouble.

Reply:
`CommunityIntroMessageInput` does not behave well on different view :man_shrugging: :
![image](https://user-images.githubusercontent.com/33099791/182125455-d1e2d721-b0ec-4161-bef0-74de97451fa2.png)